worker/test/src/RTC/TestRtpRetransmissionBuffer.cpp

```diff
@@ -28,15 +28,15 @@ class RtpMyRetransmissionBuffer : public RtpRetransmissionBuffer
 	void Insert(uint16_t seq, uint32_t timestamp)
 	{
 		// clang-format off
-		uint8_t rtpBuffer[] =
+		uint8_t rtpBuffer[1500] =
```

Comment:
Why does it need to be this large, is there a way to calculate it from some constant rather than using a magic constant here?